### DISCUSSION TITLE
The slider for changing the 'Line width' in the Painting Editor has a different style from the sliders for other features, such as Blur and Filter.

### DISCUSSION THREAD

**Discussion by thanglq1** [hm21/pro_image_editor#103] [Q&A] [answered]
Hi @hm21. The slider for changing the 'Line width' in the Painting Editor has a different style from the sliders for other features, such as Blur and Filter. If you have time to update, please help add a configuration to allow updating the text style and close button on the bottom sheet for changing the 'Line width'.
Thank you!

**Comment by hm21** [comment, marked as answer]
Sure, I released version `3.0.14` which includes this stuff. You can find the slider and the close button in `customWidgets` and the text style for the title in the `imageEditorTheme` / `paintingEditor`.

**Reply by thanglq1** [reply]
It is working now. Thank you!
